02-oop-inheritance/javaScript/refactored_JS/main  v0.1.js

```diff
@@ -0,0 +1,44 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function ( title ) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function( title ) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+var Movie = function (paramName, paramValue) {
+  //Variables
+  this.attributes = [];
+  this.attributes[paramName] = paramValue; 
```

Comment:
+1. You're trying to access an array index with (what I'm supposing ``paramName`` would be) a string.